resources/plugin_uploader/plugin_uploader.py

```diff
@@ -33,6 +35,7 @@ class Artifact(BaseModel):
     type: str
     goos: str | None = None
     goarch: str | None = None
+    extra: ArtifactExtra
```

Comment:
```suggestion
    extra: ArtifactExtra | None = None
```
maybe this will fix the broken test